packages/perseus/src/widgets/explanation.tsx

```diff
@@ -71,85 +71,75 @@ class Explanation extends React.Component<Props, State> {
-                >
-                    {() => <View style={viewStyling}>{`[${promptText}]`}</View>}
-                </Clickable>
-            );
-        }
-
-        const expandedStyle = isMobile
-            ? styles.contentExpandedMobile
-            : styles.contentExpanded;
+        const caretIcon = this.state.expanded ? caretUp : caretDown;
+
+        const allowTransition = window.matchMedia(
+            "(prefers-reduced-motion: no-preference)",
+        ).matches;
+
+        // Special styling is needed to fit the button in a block of text without throwing off the line spacing.
+        // While the button is not normally included in a block of text, it needs to be able to accommodate such a case.
+        const buttonStyleOverrides = {
+            height: "22px",
+            marginLeft: "-2px",
+            padding: "0 2px",
+        };
+
+        const contentStyling = css(
+            styles.content,
+            this.state.expanded
+                ? styles.contentExpanded
+                : styles.contentCollapsed,
+            allowTransition &&
+                (this.state.expanded
+                    ? styles.transitionExpanded
+                    : styles.transitionCollapsed),
+        );
+
+        const contentTestId =
+            process.env.NODE_ENV === "test" ? "content-container" : null;
 
         return (
-            <div className={css(styles.container)}>
-                {promptContainer}
-                {this.state.expanded && (
-                    <div
-                        className={css(
-                            styles.content,
-                            isMobile && styles.contentMobile,
-                            this.state.expanded && expandedStyle,
-                        )}
-                        // eslint-disable-next-line react/no-string-refs
-                        ref="content"
-                    >
-                        <Renderer
-                            apiOptions={this.props.apiOptions}
-                            content={this.props.explanation}
-                            widgets={this.props.widgets}
-                            linterContext={this.props.linterContext}
-                        />
-                    </div>
+            <UniqueIDProvider
+                mockOnFirstRender={false}
+                scope="explanation-widget"
+            >
+                {(ids) => (
+                    <>
+                        <Button
+                            aria-expanded={this.state.expanded}
+                            aria-controls={ids.get("content")}
+                            endIcon={caretIcon}
+                            kind="tertiary"
+                            onClick={this._onClick}
+                            size="large"
+                            style={buttonStyleOverrides}
+                        >
+                            {promptText}
+                        </Button>
+
+                        <div
```

Comment:
Today, we mostly prefer using a Wonder Blocks [`View`](https://khan.github.io/wonder-blocks/?path=/docs/core-view--docs) (which is basically a `<div />`).  This then means you don't need to use the `Aphrodite` `css()` function and can pass the styles directly to the `<View style={} />` prop.

Reply:
Also, I realize that the original content used `<div />`'s, so if this change is too dramatic, I'm ok with you skipping the change to using a `<View />`.

Reply:
`View`s differ from divs mainly by having `display: flex; flex-direction: column;` by default. This div uses `display: inline-grid`, so the View defaults get overridden. I think it's fine to use a div here.

Reply:
I'm cool with switching to `<View>` since it helps with consistency in using WB.